app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -44,54 +44,21 @@ public SearchResultHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(final SearchResultHolder holder, int position) {
-        Datum datum = datumList.get(position);
+        final Datum datum = datumList.get(position);
+        holder.previewImageView.setVisibility(View.GONE);
         if (datum != null) {
             holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
             holder.descriptionTextView.setText(Html.fromHtml(datum.getDescription()));
-            if (!TextUtils.isEmpty(datum.getLink())) {
-                LinkPreviewCallback linkPreviewCallback = new LinkPreviewCallback() {
```

Comment:
@Orbiter mentioned that for rss action type " client must not search for itself, but just shows a search result which is provided by the server" that's why I removed linkpreviewcallback. [LinkPreviewCallback](https://github.com/LeonardoCardoso/Android-Link-Preview) makes a preview from url.